.gitlab-ci.yml

```diff
@@ -225,8 +225,7 @@ deploy-prod-canary:
   stage: deploy:canary
   extends:
     - .base-configuration
-    - .tags
-  when: manual
+    - .main
   allow_failure: false
```

Comment:
allow failure should not be needed anymore

Reply:
sure